### PR TITLE
Clear docker host op

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: content/io/cloudslang/docker/maintenance/clear_docker_host.sl [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+####################################################
+# Deletes all Docker images and containers from Docker Host.
+#
+# Inputs:
+#   - docker_host - Docker machine host
```
Comment (orius123) [inline content/io/cloudslang/docker/maintenance/clear_docker_host.sl]: can;t we just call it host,username and password?

File: content/io/cloudslang/docker/maintenance/clear_docker_host.sl [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                required: false
+            - private_key_file:
+                required: false
+            - timeout:
+                required: false
+        publish:
+          - all_containers: container_list
+    - clear_all_containers:
+        do:
+          docker_containers.clear_container:
```
Comment (orius123) [inline content/io/cloudslang/docker/maintenance/clear_docker_host.sl]: does it also stops running containers?

File: content/io/cloudslang/docker/maintenance/clear_docker_host.sl [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            - docker_password:
+                required: false
+            - private_key_file:
+                required: false
+            - timeout:
+                required: false
+        publish:
+          - amount_of_images_deleted
+          - amount_of_dangling_images_deleted
+          - dangling_images_list_safe_to_delete
```
Comment (orius123) [inline content/io/cloudslang/docker/maintenance/clear_docker_host.sl]: why do we need to publish: dangling_images_list_safe_to_delete and images_list_safe_to_delete

File: content/io/cloudslang/docker/maintenance/clear_docker_host.sl (hunk 2 above)
Comment (meirwah) [inline content/io/cloudslang/docker/maintenance/clear_docker_host.sl]: yes - it does stop+remove

File: content/io/cloudslang/docker/maintenance/clear_docker_host.sl (hunk 1 above)
Comment (meirwah) [inline content/io/cloudslang/docker/maintenance/clear_docker_host.sl]: good question...
It makes it clearer that this is the Docker host params.
Let's say this flow also needs to auth with an app, so the app param should be username/password, how will you diff between this params and the docker host one's ?

for sure we need to align this..

**Comment by meirwah** [comment]
@orius123 please merge

**Pull request merged**